app/views/shows/index.html.haml

```diff
@@ -1,6 +1,16 @@
-%h1 Smartflix
-- @shows.each do |show|
-  %article
-    %h2= "#{show['title']} (#{show['release_year']})"
-    %p= show['type']
-    %p= show['description']
+%div{:class => "container row"}
+  %div{:class => "col-sm-12"}
+    %h1{:class => "display-4"} Smartflix
+    %p{:class => "lead"} Find out more about your favourite movies.
+  %div{:class => "container row"}
+    - @shows.each do |show|
+      %div{:class => "col-sm-3 d-flex align-items-stretch mb-4"}
+        %div{:class => "card"}
+          %img{:class => "card-img-top", :alt => "placeholder", :src => "https://via.placeholder.com/150"}
+            %div{:class => "card-body"}
+              %h5{:class => "card-title"}= "#{show['title']} (#{show['release_year']})"
+              %p{:class => "card-text"}= show['type']
+              %p{:class => "card-text"}= show['description']
+
+
+
```

Comment:
It may not be necessary to leave so much trailing whitespace at the end of the file – unless you have a reason for doing so of course!